File: src/test/java/no/utgdev/diy/ex6/MinComponent.java
package no.utgdev.diy.ex6;

import no.utgdev.diy.annotations.Component;
import no.utgdev.diy.annotations.Inject;

@Component
public class MinComponent {
    @Inject
    private MinService service;

    public String hei() {
        return "Hei fra MinComponent og " + service.hei();
    }
}

File: src/test/kotlin/no/utgdev/diy/ex6/MinComponent.kt
package no.utgdev.diy.ex6

//REMOVE import no.utgdev.diy.annotations.Component;
//REMOVE import no.utgdev.diy.annotations.Inject;

//REMOVE @Component
class MinComponent {
    //REMOVE @Inject
    private val service: MinService? = null
    fun hei(): String {
        return "Hei fra MinComponent og " + service!!.hei()
    }
}